alonet/deformable_detr/deformable_transformer.py

```diff
@@ -199,31 +199,42 @@ def get_valid_ratio(self, mask):
         return valid_ratio
 
     def forward(self, srcs, masks, pos_embeds, query_embed=None, **kwargs):
-        transformer_outputs = {}
-
         assert self.two_stage or query_embed is not None
 
+        transformer_outputs = {}
+
         # prepare input for encoder
         src_flatten = []
         mask_flatten = []
         lvl_pos_embed_flatten = []
-        spatial_shapes = []
+        if "is_export_onnx" in kwargs:
```

Comment:
Do we still need to use `is_export_onnx` ? Is is not supposed to be replace by is_tracking ?

Reply:
Not sure tracking = {...} is great since it is not self explained. What do you think of having 
`Detr(is_tracing)` and `Detr(is_tracing, fixed_input_shape=(h, w))` -> By default the model assume dynamic input shape. Except when exporting to onnx/tensort.  ? @Johansmm

Reply:
I renamed "is_trace" to "tracing"